source/administration/performance-alerting-guide.rst

```diff
@@ -1,7 +1,9 @@
 Mattermost Performance Alerting Guide
 ======================================
 
-Mattermost uses Prometheus and Grafana to track performance metrics of the Mattermost application servers. The purpose of this guide is to help you set up alerts on your Grafana dashboard.
+Mattermost recommends using Prometheus and Grafana to track performance metrics of the Mattermost application servers. The purpose of this guide is to help you set up alerts on your Grafana dashboard.  
+
+.. note:: We highly recommend setting up performance alerting for deployments above 5,000 users, where additional servers have been added for performance load-balancing.
```

Comment:
Why 5,000 users (and not, say, 1,000 or 10,000)?

Reply:
Great question @jasonblais - I am open to any feedback on this.  I used https://docs.mattermost.com/deployment/scaling.html#sample-scaling-guide as a resource and noted "As deployments scale above 5,000 users, additional servers are added for performance load-balancing and for providing additional redundancy (see our High Availability Cluster guide)." 

Since we had made the recommendation for performance load balancing at the 5,000 user threshold, I thought it made sense to consider adding metrics at this time.

Reply:
Yes, we have to make do with random number for now. The load tests (which are going to be out soon<sup>TM</sup>)  are going to answer this question.

Reply:
Thanks Katie & Agniva, makes sense. At some point it would be helpful to review the recommendations are consistent across the docs (and our customer/pre-sales teams)

Reply:
Totally agree. This is our highest priority right now. And hopefully we will be able to have something to stable to show soon.